src/mathfuns.jl

```diff
@@ -52,8 +54,9 @@ for (meth, libnm) in [
                       (:sqrt,:sqrt),
                       (:exp,:exp),
                       ]
-    eval(Expr(:import, :Base, meth))
-    IMPLEMENT_ONE_ARG_FUNC(:(Base.$meth), libnm)
+    modu = meth != :gamma ? :Base : :SpecialFunctions
```

Comment:
Can you make this a part of the list above? i.e. `(:abs, :abs, :Base)`? Also can you move, `eta, zeta`, from the list below to the above list?